### PR TITLE
Wait for update before reading timer count

### LINKED ISSUE #1180: Double check reading the current timer value
We need the change done in #1178 for other chips, too

### PR DESCRIPTION
**Pull request by bjoernQ**
Closes #1180 

In #1178 there was a check introduced before reading the timer count. We should do it for all chips.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix `hello_rgb` example for ESP32 (#1173)
 - Fix timer `now` for esp32c3 and esp32c6
 - Fixed the multicore critical section on Xtensa (#1175)
+- Fix timer `now` for esp32c3 and esp32c6 (#1178)
```
Comment (MabezDev) [inline CHANGELOG.md:36]:
```suggestion
```

Looks like this duplicated?

**Review by MabezDev — CHANGES_REQUESTED** [review]
Just the changelog to fix, but other than that LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (bjoernQ) [reply CHANGELOG.md:36]: oh yes this looks suspicious but that other PR didn't include the PR number in the changelog so I added it here - this is just a follow up of the first one

Comment (MabezDev) [reply CHANGELOG.md:36]: Ah sorry I picked the wrong line, could you remove line 34? That's the duplicate without the PR number

Comment (bjoernQ) [reply CHANGELOG.md:36]: ah sure ... haven't seen it 🙈

**Review by MabezDev — APPROVED** [review]

**Pull request merged**